src/Response.php

```diff
@@ -45,6 +45,16 @@ public function status(): Status
         return new Status($this->attributes['status']);
     }
 
+    /**
+     * Get the response message.
+     *
+     * @return Message
+     */
+    public function message(): Message
```

Comment:
Are you sure this is on every response? Could you check some of the BusinessEmailOrders. If it's not can you move this into the `AddResponse` class etc.

Reply:
Started in the AddResponse and UpdateResponse classes, but noticed the duplication so moved it out here.

From my testing, each response has a `status` attribute and `msg`

Reply:
I would just double check some of the other responses from other endpoints because if we put it at that level we stating to people using this repo that there should be a message.

Reply:
It's new functionality, so if it doesn't work people will get an error and report it. No one is using ->message() right now (apart from me this afternoon :P)

Do you think it is in any way backwards INcompatible?

(I am struggling for time to test every previous case manually, but my sample of tests have all passed).